spec/spec_helper.rb

```diff
@@ -11,6 +11,12 @@
 
 require_relative 'core_ext/string'
 
+begin
+  require 'pry'
+rescue LoadError
+  # pry is not activated
```

Comment:
This is a nitpick :-)

```suggestion
  # Pry is not activated.
```

Reply:
Fixed!